### ISSUE TITLE
High ID Numbers Encoded

### ISSUE THREAD

**Issue by myst4** [Pandora-Labs-Org/erc404#38]
Hello,

I've just tested the new version and it's working very well. However, I have a question regarding the ID encoding system.

I've noticed that the ID encoding system generates quite high numbers. I understand there might be technical reasons behind this decision, but I'd like to know if it's absolutely necessary to keep these values so high.

Would it be possible to configure or edit the encoding system to start the ID sequence from a lower number, such as 1, to have IDs like 1, 2, 3, 4, etc.? This would make management and tracking much easier for my specific use case.

Thank you for your attention, and I look forward to your response.

**Comment by nicezic** [comment]
Yes, it is big bothering.

**Comment by CrunchNZ** [comment]
Yes and without any documentation around why it's done, how to build around it and why it's necessary at all, it is super frustrating to work with. No explanation around it makes it infuriating to build with.
